test/e2e/apps/job.go

```diff
@@ -510,17 +511,18 @@ done`}
 		framework.ExpectNoError(err, "failed to get latest job object")
 		gomega.Expect(job.Status.Active).Should(gomega.Equal(int32(0)))
 		gomega.Expect(job.Status.Ready).Should(gomega.Equal(ptr.To[int32](0)))
-		gomega.Expect(job.Status.Terminating).Should(gomega.Equal(ptr.To[int32](0)))
+		// TODO (https://github.com/kubernetes/enhancements/issues/3939): Restore the assert on .status.terminating when PodReplacementPolicy goes GA.
```

Comment:
Can you leave that assertion here, so it'll be easier to get it back once that other enhancement goes GA?

Reply:
That sounds great to me. Thank you.

Reply:
Done.